### PR TITLE
MAT-6489 mapping matMeasure.details.definition to madieMeasure.metaDa…

### PR DESCRIPTION
**Pull request by RohitKandimalla**
…ta.measureDefinitions

## PR Template

Jira Ticket: [MAT-6489](https://jira.cms.gov/browse/MAT-6489)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e. Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e. Changing auth methods, Adding a new user type, Adding a required but vulnerable package)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: lambda-function/src/utils/measureConversionUtils.ts [hunk 1]
```diff
@@ -118,7 +118,9 @@ const convertMeasureMetadata = (measureDetails: MeasureDetails): MeasureMetadata
     clinicalRecommendation: measureDetails.clinicalRecomms,
     references: references,
     endorsements: [endorsement],
-    definition: measureDetails.definitions,
+    measureDefinitions: !_.isEmpty(measureDetails.definitions)
+      ? [{ term: "", definition: measureDetails.definitions || "" }]
```
Comment (adongare) [inline lambda-function/src/utils/measureConversionUtils.ts]: should we create an `id` as well either here or in the transfer controller/service?

Comment (RohitKandimalla) [reply lambda-function/src/utils/measureConversionUtils.ts:122]: I was thinking of adding it to transfer controller, Since we have models already. I will create a PR for that in few minutes

File: lambda-function/package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "@types/node": "^17.0.13",
     "@types/nodemailer": "^6.4.14",
     "@types/nodemailer-smtp-transport": "^2.7.8",
+    "@types/uuid": "^9.0.8",
```
Comment (adongare) [inline lambda-function/package.json]: we can use nodejs `crypto` so probably no need to add new package. we are using it for generating population IDs I think already  https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID

**Review by adongare — DISMISSED** [review]

File: lambda-function/package.json (hunk 2 above)
Comment (RohitKandimalla) [reply lambda-function/package.json]: I see that, updated to use Crypto instead of a new package

**Review by adongare — APPROVED** [review]
